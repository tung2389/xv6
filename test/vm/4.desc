Test write after protect on child