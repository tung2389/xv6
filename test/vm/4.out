pid 4 test_4: trap 14 err 7 on cpu 0 eip 0x104b addr 0x1000--kill proc
